pixl_dcmd/src/pixl_dcmd/_database.py

```diff
@@ -100,6 +119,7 @@ def get_unexported_image(
                 Extract.slug == project_slug,
                 Image.study_uid == study_info.study_uid,
                 Image.exported_at == None,  # noqa: E711
+                Image.pseudo_patient_id == study_info.pseudo_patient_id,
```

Comment:
The pseudo patient ID isn't going to be in parquet files so we shouldn't query on it. Expect that this wouldn't match in any case:
- if it is from Ion DRI then the pseudo id in the database was set, the DICOM will have the hashed identifier from anonymisation
- if the database has no pseudo id, then it'll be none, but the DICOM will have the hashed identifier from anonymisation